src/components/sdk/DeviceSelection/CameraSelection/index.tsx

```diff
@@ -14,25 +13,18 @@ interface Props extends BaseSdkProps {
   notFoundMsg?: string;
   /** The label that will be shown for camera selection, it defaults to "Camera source". */
   label?: string;
-  /** A boolean that determines whether or not to include additional sample video devices, such as "None", "Blue", "SMTP Color Bars". Defaults to true. This will be deprecated in the next major version. */
-  appendSampleDevices?: boolean;
 }
 
 export const CameraSelection: React.FC<Props> = ({
   notFoundMsg = 'No camera devices found',
   label = 'Camera source',
-  appendSampleDevices = true,
   ...rest
 }) => {
   const meetingManager = useMeetingManager();
-  const videoInputConfig: DeviceConfig = {
-    additionalDevices: appendSampleDevices,
+  const { devices, selectedDevice } = useVideoInputs();
+  const selectVideoInput = async (deviceId: string): Promise<void> => {
```

Comment:
Should this `deviceId: string` be `VideoInputDevice` type instead? I know we wont need a transform device here but we also do not want just the `stringId` right?

Reply:
Yeah, I'm working on it in a separate PR. This PR is for the first task in design doc - Adapt to  JS SDK Device Controller V3 Changes. Rest of tasks will be in another PR.

Reply:
As discussed in standup, I will not raise another PR but push commits to this one. So this change will be in later.

Reply:
Got it!